src/shared/install.h

```diff
@@ -29,14 +29,14 @@ enum UnitFilePresetMode {
 /* This enum type is anonymous, since we usually store it in an 'int', as we overload it with negative errno
  * values. */
 enum {
-        UNIT_FILE_SYMLINK,
-        UNIT_FILE_UNLINK,
-        UNIT_FILE_IS_MASKED,
-        UNIT_FILE_IS_DANGLING,
-        UNIT_FILE_DESTINATION_NOT_PRESENT,
-        UNIT_FILE_AUXILIARY_FAILED,
-        _UNIT_FILE_CHANGE_TYPE_MAX,
-        _UNIT_FILE_CHANGE_TYPE_INVALID = -EINVAL,
+        INSTALL_CHANGE_SYMLINK,
+        INSTALL_CHANGE_UNLINK,
+        INSTALL_CHANGE_IS_MASKED,
+        INSTALL_CHANGE_IS_DANGLING,
+        INSTALL_CHANGE_DESTINATION_NOT_PRESENT,
+        INSTALL_CHANGE_AUXILIARY_FAILED,
+        _INSTALL_CHANGE_MAX,
+        _INSTALL_CHANGE_INVALID = -EINVAL,
```

Comment:
there probably should really `_INSTALL_ERRNO_MAX = -ERRNO_MAX,` here so that we cover the full error range. we do that at other places